src/ProtocolGateway.Core/Instrumentation/WindowsPerformanceCounterManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using Microsoft.Azure.Devices.ProtocolGateway.Instrumentation;
+
+    public class WindowsPerformanceCounterManager : IPerformanceCounterManager
```

Comment:
then there should be no #ifs in this file

---
In reply to: [114228316](https://github.com/ancaantochi/azure-iot-protocol-gateway/pull/1#discussion_r114228316) [](ancestors = 114228316)